src/ceph-daemon

```diff
@@ -1062,6 +1062,8 @@ def command_bootstrap():
         ).run()
 
         logger.info('Adding key to root@localhost\'s authorized_keys...')
+        if not os.path.exists('/root/.ssh'):
+            os.mkdir('/root/.ssh')
```

Comment:
need to also set the mode to 0700 for ``.ssh``, ``os.mkdir('/root/.ssh', mode=0o700)``

Reply:
fixed and repushed.